src/test/unit_tests/jest_tests/test_nc_nsfs_schema_validation.test.js

```diff
@@ -407,7 +413,7 @@ describe('schema validation NC NSFS account', () => {
                 email: account_email,
                 access_keys: [
                     {
-                    // hide it on purpose
```

Comment:
didn't get why we need this comment?

Reply:
@naveenpaul1 I decided to comment out the code lines so it will be clear that the object is sent without the required properties.

Reply:
@naveenpaul1 I refactored the file, I hope it will be more convenient.